app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -726,18 +691,6 @@ def build_schedule_options_for_select
     ]
   end
 
-  def retrieve_aws_regions
-    ManageIQ::Providers::Amazon::Regions.regions.flat_map { |region| [region[1].values_at(:description, :name)] }
-  end
-
-  def retrieve_openstack_api_versions
-    [['Keystone v2', 'v2'], ['Keystone v3', 'v3']]
-  end
-
-  def retrieve_security_protocols
-    [[_('SSL without validation'), 'ssl'], [_('SSL'), 'ssl-with-validation'], [_('Non-SSL'), 'non-ssl']]
-  end
```

Comment:
These 3 methods were orphaned in https://github.com/ManageIQ/manageiq-ui-classic/pull/7943

Reply:
So what's weird is I see that the FileDepot subclasses need it, but I don't know what's setting it - I have a feeling the PxeServer create/validate page is broken (I found other bugs in there anyway).